localization/english/culture/ctp_titles_l_english.yml

```diff
@@ -414,6 +414,144 @@
  count_tribal_female_italian:0 "Capo"
  duke_tribal_female_italian:0 "Gran Capo"
 
+ ##Roman##
+ baron_male_roman:0 "Baron"
```

Comment:
Almost certainly a mistake in Master Sheet.

Reply:
You can edit any localisations directly by yourself which are wrong. 
I Just added in what was already present in the sheets and have absolutely no knowledge on my own regarding this, so I give you full clearance to go bat shit crazy on my localisations file